pom.xml

```diff
@@ -13,18 +12,6 @@
   <packaging>hpi</packaging>
   <name>Workspace Rotation Plugin</name>
   <url>http://wiki.hudson-ci.org/display/HUDSON/Workspace+Rotation+Plugin</url>  
-  <build>
```

Comment:
@jglick Now, there is a wiki page https://wiki.jenkins-ci.org/display/JENKINS/Workspace+Rotation+Plugin